### ISSUE TITLE
HDF5 tests fail due to hardcoded paths

### ISSUE THREAD

**Issue by pgrete** [parthenon-hpc-lab/parthenon#255]
If the bin directory is located outside of the Parthenon root directory, the hdf5 tests fail as 
- the `utils` is not found (not in default PYTHONPATH)
- the path to the hdf5 diff script is hardcoded

**Comment by JoshuaSBrown** [comment]
Do you have a solution for this? I did not know how else to include a python module that was in a separate folder hierarchy?

**Comment by Yurlungur** [comment]
The python scripts that run the test knows the path of itself, right. Can it not use that to find where the `utils` module is?

**Comment by JoshuaSBrown** [comment]
> The python scripts that run the test knows the path of itself, right. Can it not use that to find where the `utils` module is?

@pgrete why are you wanting to move the bin outside of the parthenon root directory? Are you trying to pull it out of the repository? Just trying to understand.

**Comment by pgrete** [comment]
> The python scripts that run the test knows the path of itself, right. Can it not use that to find where the `utils` module is?

Yes, that's the intended fix. The hdf5 diff python tool is already using a method to insert a path to find the right module.

**Comment by pgrete** [comment]
> > The python scripts that run the test knows the path of itself, right. Can it not use that to find where the `utils` module is?
> 
> @pgrete why are you wanting to move the bin outside of the parthenon root directory? Are you trying to pull it out of the repository? Just trying to understand.

Testing on Summit forced me to do this ;)
The home directories are not writable from compute nodes. Thus, in order to run the test and store the results/logs the build directory including the test directory had to be on the scratch filesystem.

**Comment by Yurlungur** [comment]
Ping @pgrete (also see issue on Crusher)
